pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -447,9 +449,9 @@ func (s *Summarizer) loadControls() error {
 				s.groupWrappersMap[g.ID] = gw
 			}
 			for _, check := range g.Checks {
-				if !check.Scored {
-					continue
-				}
+				//if !check.Scored {
```

Comment:
Do we want to leave this commented or just remove it entirely?

Reply:
this will be removed entirely